src/createSliderWithTooltip.jsx

```diff
@@ -9,11 +9,13 @@ export default function createSliderWithTooltip(Component) {
       tipFormatter: PropTypes.func,
       handleStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.arrayOf(PropTypes.object)]),
       tipProps: PropTypes.object,
+      getTooltipContainer: PropTypes.func,
     };
     static defaultProps = {
       tipFormatter(value) { return value; },
       handleStyle: [{}],
       tipProps: {},
+      getTooltipContainer: node => node.parentNode,
```

Comment:
Can we reverse https://github.com/ant-design/ant-design/pull/20699/files in favor of this PR?

Reply:
We'd better not since use `getTooltipContainer` will beak overflow hidden. [#20699](https://github.com/ant-design/ant-design/pull/20699) is aim to compatible with origin behavior